webapp/views/partials/home.jade

```diff
@@ -0,0 +1,8 @@
+div(class='container')
+    h1 {{userName}}
+
+    div(class='div-center')
```

Comment:
text-center only centers text is seems. This div-center I made centers everything inside the div, or at least the button I made.